pkg/services/user/password.go

```diff
@@ -8,8 +8,8 @@ import (
 )
 
 var (
-	ErrPasswordTooShort       = errutil.NewBase(errutil.StatusBadRequest, "password-policy-too-short", errutil.WithPublicMessage("New password is too short"))
-	ErrPasswordPolicyInfringe = errutil.NewBase(errutil.StatusBadRequest, "password-policy-infringe", errutil.WithPublicMessage("New password doesn't comply with the password policy"))
+	ErrPasswordTooShort       = errutil.BadRequest("password.password-policy-too-short", errutil.WithPublicMessage("New password is too short")).Errorf("new password is too short")
```

Comment:
"ErrorF" should probably be called when returning errors. Then it can be used to wrap other error etc

Reply:
It has been changed as requested.